.github/workflows/bump_stdlibs.yml

```diff
@@ -0,0 +1,25 @@
+name: bump_stdlibs
+
+on:
+  schedule:
+    - cron: '00 * * * *'
```

Comment:
How about weekly instead? Or every second/third day or something?

Reply:
I've changed it to run three times a week (00:00 UTC on Mondays, Wednesdays, and Fridays). How does that sound?

I guess we could make it twice a week if you want? One time per week seems a little too infrequent - Pkg can certainly accumulate a lot of new commits in seven days.

Reply:
> One time per week seems a little too infrequent - Pkg can certainly accumulate a lot of new commits in seven days.

I think one time a week is very much sufficient.

Reply:
Yea, if there are urgent fixes one can manually update, this is mostly just so we don't forget to do it at all during a month etc. I think it will quickly be annoying if we get bump PRs too often.

Reply:
Sounds good. I’ve changed it to run once per week (00:00 UTC every Monday)